lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -572,6 +572,18 @@ MITIGATION_KBV_FAIL_M2B:
       targetJourney: INELIGIBLE
       targetState: INELIGIBLE
 
+PYI_ESCAPE_M2B:
+  response:
+    type: page
+    pageId: pyi-escape-m2b
+  events:
+    next:
+      targetState: IPV_IDENTITY_START_PAGE
+    bankAccount:
+      targetState: BANK_ACCOUNT_START_PAGE
+    end:
+      targetState: END
```

Comment:
These might need tweaking
- `IPV_IDENTITY_START_PAGE` -> `IDENTITY_START_PAGE`
- `END` -> `RETURN_TO_RP` (assuming we don't need to go to TICF in this case, but I don't think we do, as they won't have visited any CRIs)